src/parser.jl

```diff
@@ -2722,11 +2722,18 @@ function parse_cat(ps::ParseState, closer, end_is_symbol)
                     whitespace_newline=false,
                     for_generator=true)
     k = peek(ps, skip_newlines=true)
+    mark = position(ps)
     if k == closer
         # []  ==>  (vect)
         return parse_vect(ps, closer)
+    elseif k == K";"
+        # [;;]      ==>  (ncat 2)
+        # [;; \n ]  ==>  (ncat 2)
```

Comment:
I'll fix these separately to keep them in sync with the actual tests (yeah, I need to improve the tooling so we don't need to repeat ourselves :sweat_smile: )

Reply:
Oops! Yeah, I think I was just thinking of the s-expression syntax instead of JuliaSyntax' one.